inc/admin.php

```diff
@@ -159,7 +163,7 @@ function settings_screen() {
 function format_error( $error ) {
 	/* translators: Error message output. 1: Line number, 2: Error message */
 	$message = sprintf(
-		__( 'Line %1$s: %2$s', 'ads-txt' ),
+		esc_html__( 'Line %1$s: %2$s', 'ads-txt' ),
```

Comment:
It's possible for the second placeholder to contain HTML content (see other comments).

Reply:
could we add the escaping the the values used by sprintf on the lines below ?